contrib/depends/packages/sodium.mk

```diff
@@ -1,8 +1,8 @@
 package=sodium
-$(package)_version=1.0.16
-$(package)_download_path=https://github.com/jedisct1/libsodium/archive
-$(package)_file_name=$($(package)_version).tar.gz
-$(package)_sha256_hash=0c14604bbeab2e82a803215d65c3b6e74bb28291aaee6236d65c699ccfe1a98c
+$(package)_version=1.0.15
```

Comment:
Any particular reason to downgrade libsodium ?

Reply:
I could not get that version to play nice with the darwin build. If it is a problem, and there is a reason for taking 1.0.16 , I'll try to get it running.